integration/test_db.py

```diff
@@ -25,11 +26,35 @@
 from tests import testutils
 
 
+def integration_conf(request):
+    host_override = os.environ.get('FIREBASE_DATABASE_EMULATOR_HOST')
+    if host_override:
+        return None, 'fake-project-id'
+    else:
+        return conftest.integration_conf(request)
+
+
+@pytest.fixture(scope='module')
+def app(request):
+    cred, project_id = integration_conf(request)
+    ops = {
+        'databaseURL' : 'https://{0}.firebaseio.com'.format(project_id),
+    }
+    return firebase_admin.initialize_app(cred, ops, name='integration-db')
+
+
+@pytest.fixture(scope='module', autouse=True)
```

Comment:
Why is this necessary?

Reply:
Otherwise, the `default_app` in `conftest.py` will run (because it's `autouse`) and fail even if we only run the db integration tests with the emulator.